### PR TITLE
GEOMESA-972 Remove DataStore Edit Panels

### PR DESCRIPTION
**Pull request by jnh5y**
* Removes datastore panels.
* Adds some datastore parameters to the getParametersInfo.

Signed-off-by: Jim Hughes <jnh5y@ccri.com>

### CODE REVIEW

**Comment by jdk2pq** [comment]
Passwords are still plaintext fields for the Accumulo (GeoMesa) vector data store

**Comment by jnh5y** [comment]
@jdk2pq Thanks for the catch!  @anthonyccri don't merge:)

**Comment by jdk2pq** [comment]
Looks good to me! Tested and working. The old version of the datastore also has:
- writeThreads
- queryThreads
- recordThreads
- collectStats
- caching
for connection params. Do we want to keep those?

File: geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreFactory.scala [hunk 1]
```diff
@@ -10,13 +10,13 @@
...
 import java.io.Serializable
-import java.util.{Map => JMap}
+import java.util.{Map => JMap, Collections}
 
 import org.apache.accumulo.core.client.mock.{MockConnector, MockInstance}
 import org.apache.accumulo.core.client.security.tokens.{AuthenticationToken, PasswordToken}
 import org.apache.accumulo.core.client.{Connector, ZooKeeperInstance}
 import org.geotools.data.DataAccessFactory.Param
-import org.geotools.data.DataStoreFactorySpi
+import org.geotools.data.{Parameter, DataStoreFactorySpi}
```
Comment (jdk2pq) [inline geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreFactory.scala]: organize imports

**Comment by elahrvivaz** [comment]
@jdk2pq @jnh5y those options are no longer present? we want to keep those.

**Comment by jdk2pq** [comment]
The Kafka Data Store now has the zkPath automatically populated with "/geomesa/ds/kafka". Don't know if that's expected or not.

**Comment by jdk2pq** [comment]
We have both an "Accumulo Coverage Format" and a "GeoMesa Coverage Format". I'm not sure of the differences between the two, but we should probably only have one. @AndrewAnnex, any input?

**Comment by jdk2pq** [comment]
EDIT: Whoops, collectStats and caching are still there, now just checkboxes instead of input fields.

**Comment by elahrvivaz** [comment]
Is this PR ready to merge?

**Comment by elahrvivaz** [comment]
e775ca2895f574264c05079dbac0d19e3490391a

**Pull request closed without merging**